kt/plugins/godot-intellij-plugin/src/main/kotlin/godot/intellij/plugin/inspections/registration/quickfix/EngineFunctionNotRegisteredQuickFix.kt

```diff
@@ -0,0 +1,19 @@
+package godot.intellij.plugin.inspections.registration.quickfix
```

Comment:
Name and file and class should not change to notification ?

Reply:
Ups. Cood catch. I'll change it.